apps/microtvm/zephyr/demo_runtime/qemu-hack/qemu-system-riscv64

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
```

Comment:
I thiiiink you can just checkin a symlink here and `$(basename $0)` will take care of this. could you try it out?

e.g. `ln -s qemu-system-i386 qemu-system-riscv64` in this dir.

Reply:
it worked!